web/ui_utils.js

```diff
@@ -126,7 +126,9 @@ function scrollIntoView(element, spot, scrollMatches = false) {
   while (
     (parent.clientHeight === parent.scrollHeight &&
       parent.clientWidth === parent.scrollWidth) ||
-    (scrollMatches && parent.classList.contains("markedContent"))
+    (scrollMatches &&
+      (parent.classList.contains("markedContent") ||
+        getComputedStyle(parent).overflow === "hidden"))
```

Comment:
The following could possibly work as well, and would likely be more efficient than a `getComputedStyle` call, but with the upcoming release we probably shouldn't take unnecessary risks here.
```suggestion
        parent.classList.contains("textLayer")
```